src/octuple.ts

```diff
@@ -426,4 +465,40 @@ export {
   useMaxVisibleSections,
   useOnClickOutside,
   useScrollLock,
+  // Group exported locales separately for readability.
```

Comment:
to keep this cleaner, would it be possible to intentionally export this from `@eightfold.ai/octuple/locales`  or similar?

Reply:
Would need to create a new octuple directory for that. Figure the lower the bar for import the better over a redundant schema.

Reply:
Done